server/chat-plugins/mafia.ts

```diff
@@ -3774,6 +3782,62 @@ export const commands: Chat.ChatCommands = {
 			this.parse('/mafia hostbanhelp');
 		},
 
+		gamebanname: 'namehostban',
+		namegameban: 'namehostban',
+		hostbanname: 'namehostban',
+		namehostban(target, room, user, connection, cmd) {
+			if (!target) return this.parse('/help mafia namehostban');
+			room = this.requireRoom();
+			this.checkCan('warn', null, room);
+
+			const [targetUser, rest] = this.splitOne(target);
+			const [string1, string2] = this.splitOne(rest);
+			let duration, reason;
+			if (parseInt(string1)) {
+				duration = parseInt(string1);
+				reason = string2;
+			} else {
+				duration = parseInt(string2);
+				reason = string1;
+			}
```

Comment:
So were allowing the command's reason and duration to be swapped as desired? Could that get confusing/be problematic at any point for users?

Reply:
This was mostly done to be consistent with current non-name host/gameban design